src/command_modules/azure-cli-network/azure/cli/command_modules/network/custom.py

```diff
@@ -76,15 +76,23 @@ def _upsert(parent, collection_name, obj_to_add, key_name):
     collection.append(obj_to_add)
 
 
-def _get_default_value(balancer, property_name, option_name):
+def _get_default_name(balancer, property_name, option_name):
+    return _get_default_value(balancer, property_name, option_name, True)
+
+
+def _get_default_id(balancer, property_name, option_name):
+    return _get_default_value(balancer, property_name, option_name, False)
+
+
+def _get_default_value(balancer, property_name, option_name, name):
```

Comment:
Maybe `use_name` or something instead of `name`?
When I see `name` I expect a string not bool.